include/exadg/incompressible_navier_stokes/spatial_discretization/spatial_operator_base.h

```diff
@@ -613,7 +612,10 @@ class SpatialOperatorBase : public dealii::Subscriptor
   initialize_operators(std::string const & dof_index_temperature);
 
   void
-  initialize_turbulence_model();
+  initialize_viscosity_model();
+
+  void
+  set_constant_viscosity(Number const & constant_coefficient) const;
```

Comment:
Could you check again whether this function is called more than once? If not, then we can simply write `viscous_kernel.set_constant_coefficient(viscous_kernel_data.viscosity);` at the place where this function is called currently.